client/components/EditReview.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import {ReviewForm} from '../components'
+import {updateReviewById} from '../store'
+import {connect} from 'react-redux'
+
+class EditReview extends React.Component {
+  submit = editedReview => {
+    this.props.updateReviewById(editedReview)
+    this.props.history.push(`/review/${this.props.match.params.reviewId}`)
+  }
+  render() {
+    return <ReviewForm {...this.props} onSubmit={this.submit} />
+  }
+}
+
+const mapDispatchToProps = dispatch => ({
+  updateReviewById: review => dispatch(updateReviewById(review))
+})
+
+export default connect(null, mapDispatchToProps)(EditReview)
+
+
```

Comment:
This code looks fine, but editing doesn't appear to be working when I try to test it?  It looks like the route isn't wired up?

Reply:
The route was commented out. I uncommented it.